proarc-webapp/src/main/java/cz/cas/lib/proarc/webapp/client/widget/workflow/WorkflowJobView.java

```diff
@@ -243,11 +256,21 @@ public void performAction(ActionEvent event) {
         Action addSubjobAction = Actions.emptyAction(i18n.WorkflowJob_View_NewSubjobAction_Title(),
                 "[SKIN]/actions/add.png", i18n.WorkflowJob_View_NewSubjobAction_Hint());
 
+        Action createNewObject = Actions.emptyAction(i18n.DigitalObjectCreator_FinishedStep_CreateNewObjectButton_Title(),
+                "[SKIN]/actions/save.png",
```

Comment:
Není tam lepší ikona pro přidání? V editoru vazeb je `[SKIN]/actions/add.png`. Pokud to s něčím nekoliduje. Bylo by to konzistentní.

Reply:
Já bych tam tuto ikonu nechal, stejná je pravě při zakládání objektu v částí Proarc a dělá to něco velmi podobného.

Ikona add je navíc použitá u vedlejší akce (nový záměr).

Reply:
Dobře.